README.md

```diff
@@ -1 +1,39 @@
-# Rails-Blog-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Rails Blog App
+
+> In this project, I created all controllers and views for my Blog app. I added simple placeholders for my views with plain HTML.
+
+## Built With
+
+- Ruby.
+- Rails.
+- PostgreSQL.
+- Linters.
+
+## Install
+
+- Install Ruby based on your OS from [HERE](https://www.ruby-lang.org/en/downloads/).
+- Install Rails by running this order in your terminal `gem install rails` (good luck with that).
+
+## Setup
+
+1. Clone the project using git-bash or Githup Desktop.
+2. Open the project folder with VSCode or any Editor.
+3. Open terminal and navigate to the project folder.
+4. Type `rails s` to run live server.
+5. Open your browser on this " [127.0.0.1:3000](127.0.0.1:3000) ".
+
```

Comment:
- [ ] Kindly provide all instructions on how to run this project locally(`installing bundle, setting up the database`)